### ISSUE TITLE
More accurate preview from sponsorblock by changing playback speed 

### ISSUE THREAD

**Issue by Marjo-bae** [revanced/revanced-patches#545]
### Application

YouTube Revanced

### Issue

To get more accurate preview when doing sponsorblock submission from YouTube revanced so it will make it easier for the contributors to make better quality segments while the users of sponsorblock get the benefits of better quality segments.

### Patch

Let the user set custom playback speed value such as 0.85x, 0.80x or even better to any value.

### Motivation

The preview from sponsorblock is not accurate. 
When we set at normal playback speed or 1x, the preview you get is late while the opposite happen when the playback speed set at 0.75x. Based on that, the right value to get accurate preview must be higher than 0.75x but lower than 1x. 

So by allowing users to set custom playback speed it is possible to get much more accurate preview that will result in better quality segment submissions and much more easier for contributors to make good quality segments that will improve users experience.

**Comment by oSumAtrIX** [comment]
SponsorBlock already included a UI to go frame by frame. In combination with a slow playback speed why does this not suffice?

**Comment by Marjo-bae** [comment]
Sorry for late reply. While the frame by frame is more than enough for cutting visual change. However when it comes to cut audio or speech, even at 0.75x playback speed is not good enough

**Comment by oSumAtrIX** [comment]
Implemented in commit b1e423d9cdca51e54c154cc39a24c508ca322f60.

Available patch options:
```toml
['custom-playback-speed']
granularity = "11"
min = "0.25"
max = "5.0"
```

**Comment by Marjo-bae** [comment]
Thank you for implementing this feature 👍
